test/postgresql.test.js

```diff
@@ -658,6 +658,67 @@ describe('postgresql connector', function() {
       });
     });
   });
+
+  context('json data type', function() {
+    var Customer;
+
+    before(function(done) {
+      db = getDataSource();
+
+      Customer = db.define('Customer', {
+        address: {
+          type: 'object',
+          postgresql: {
+            dataType: 'json',
+          },
+        },
+      });
+
+      db.automigrate(function(err) {
+        if (err) return done(err);
+        Customer.create([{
+          address: {
```

Comment:
Can we add test/s for 1 more level of nested keys (for checking the recursive calls in https://github.com/strongloop/loopback-connector-postgresql/pull/304/files#diff-ac54005f9bbf753aa45ceebc4a032628R341, or would that be redundant? I.e.
```
Customer.create([{
  name: {
    fullName: {
      firstName: 'foo'
      lastName: 'bar'
    }
  }
}], function (err, customers) {
...
````

Reply:
Definitely not redundant since it uncovered a bug 😅 

Syntax for deeply nested needed to be `foo->bar->>baz` since `->>` returns the value at the path as text but `->` returns the actual object at the path.

https://www.postgresql.org/docs/9.5/static/functions-json.html

Reply:
Ah great! thank you for the link :-).